flink-runtime/src/main/scala/org/apache/flink/runtime/taskmanager/TaskManager.scala

```diff
@@ -596,6 +596,10 @@ object TaskManager {
       opt[String]("tempDir") optional() action { (x, c) =>
         c.copy(tmpDir = x)
       } text ("Specify temporary directory.")
+
+      opt[String]("defaultJobManagerAdd") optional() action { (x, c) =>
```

Comment:
I'm slightly in favour of writing the option completely out: defaultJobManagerAddress